.buildkite/bk.integration.pipeline.yml

```diff
@@ -311,6 +327,116 @@ steps:
           machineType: "n1-standard-8"
           image: "${IMAGE_RHEL_8}"
 
+
+  - group: "FRH:FIPS"
+    key: integration-tests-fips
+#    depends_on:
+#      - integration_frh # FIXME: uncomment once integration_frh and frh_cleanup steps are implemented
+    steps:
+      - label: "x86_64:non-sudo: {{matrix}}"
+        depends_on: packaging-ubuntu-x86-64
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-x86-64-fips'
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} false true
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "${IMAGE_UBUNTU_2404_X86_64}" # FIXME: replace with FIPS-configured image when one is available
+        matrix:
+          - default
+
+      - label: "x86_64:sudo: {{matrix}}"
+        depends_on:
+          - packaging-ubuntu-x86-64
+        # due to deb group present in matrix tar.gz and deb packages artifacts are required
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-x86-64-fips'
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} true true
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        retry:
+          automatic:
+            limit: 1
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "${IMAGE_UBUNTU_2404_X86_64}" # FIXME: replace with FIPS-configured image when one is available
+        matrix:
+          - default
+          - upgrade
+          - upgrade-flavor
+          - standalone-upgrade
+          - fleet
+          - fleet-endpoint-security
+          - fleet-airgapped
+          - fleet-airgapped-privileged
+          - fleet-privileged
+          - fleet-upgrade-to-pr-build
+          - install-uninstall
+          - fqdn
+          - deb
+          - container
+
+      - label: "arm:sudo: {{matrix}}"
+        depends_on:
+          - packaging-ubuntu-arm64
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-ubuntu-arm64-fips'
+          .buildkite/scripts/steps/integration_tests_tf.sh {{matrix}} true true
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        agents:
+          provider: "aws"
+          image: "${IMAGE_UBUNTU_2404_ARM_64}" # FIXME: replace with FIPS-configured image when one is available
```

Comment:
IIUC, we have a FIPS image available for fleet-server called `platform-ingest-fleet-server-ubuntu-2204-fips`, I wonder if you need a similar kind of VM image, if so, the issue `5264` in the `elastic/ingest-dev` repository is the one that requested it. In case it helps.

Reply:
Yep, waiting on that private issue to be resolved (hence all the commented out bits and FIXMEs in this PR).  Just didn't want to link to it from the public repo.  Thanks.